src/functions/template-tags/general.php

```diff
@@ -1669,4 +1669,60 @@ function tribe_separated_field( $body, $separator, $field ) {
 
 		return $field ? $body_and_separator . $field : $body;
 	}
+
+	/**
+	 * Function to test if we are on the front page of the events, as WordPress language front page is different from
```

Comment:
This is good - but sometimes the information is easier to digest if we start with a nice and short sentence explaining very briefly what the function does - with any further explanations, notes or examples into additional paragraphs. For instance:

```
/**
 * Tests if we are on the site homepage and if it is set to display the main events page.
 *
 * <notes about the differences with WordPress's definition of front page, examples of
 * real usage etc>
 * 
 * @since TBD
 *
 * @return bool
 */
```

Note also the line break between 'meta' information such as `@since` and the `@return` tag.

Reply:
Thanks for the heads up.